barney.yaml

```diff
@@ -1,39 +1,32 @@
 images:
-  build-floor-alpine:
-    units:
-      - image: barney.ci/alpine%pkg/alpine-base
-      - image: barney.ci/alpine%network
-      - image: barney.ci/alpine%pkg/busybox
-      - image: barney.ci/alpine%pkg/coreutils
-
+  # Deprecated ref
   minimal-source:
+    no-create-mountpoints: true
     units:
-      - floor: .%build-floor-alpine
-        build: |
+      - build: |
           mkdir -p /dest/usr/lib/python3.11/site-packages/pybsn/
           cp pybsn/__init__.py /dest/usr/lib/python3.11/site-packages/pybsn/
 
   package:
+    no-create-mountpoints: true
     units:
-      - floor: .%build-floor-alpine
-        build: |
+      - build: |
           mkdir -p /dest/pybsn
           cp pybsn/__init__.py /dest/pybsn/
 
   test/package:
     units:
-      - floor:
-        sources: []
+      - sources: []
         mappings:
           /src/package: .%package
         build: |
           ls /src/package/pybsn/__init__.py
 
```

Comment:
That doesn't work because you no longer get the default floor and nothing except "package" is in it.

https://bsy.infra.corp.arista.io/job/J1ad233cd34697feee66db6482f3afc417b3cc58aff6b7335347c586655f0e4fe/log

Reply:
ah, good call.

Reply:
I had tried to do that previously.